cli/flox/doc/services/manifest.toml.md

```diff
@@ -0,0 +1,480 @@
+    calling `flox services stop` or after exiting the last activation of the
+    environment.
+
+`shutdown.command`
+:   A command to run to shut down the service instead of delivering the SIGTERM
+    signal to the process. Some programs require special handling to shut down
+    properly e.g. a program that spawns a server process and uses a client to
+    tell the server to shut down. Sending a SIGTERM to a client in that case
+    may not shut down the server. In those cases you may provide a specific
+    shutdown command to run instead of relying on the default behavior of
+    sending a SIGTERM to the service. This field is required if the `is-daemon`
+    field is `true`.
+
+## `[options]`
+
+The `[options]` section of the manifest details settings for the environment
+itself.
+
+The full set of options are listed below:
+```
+Options ::= {
+  systems                   = null | [<STRING>, ...]
+, allow                     = null | Allows
+, semver                    = null | Semver
+, cuda-detection            = null | <BOOL>
+}
+
+Allows ::= {
+  unfree   = null | <BOOL>
+, broken   = null | <BOOL>
+, licenses = null | [<STRING>, ...]
+}
+
+Semver ::= {
+  allow-pre-releases = <BOOL>
+}
+```
+
+`systems`
+:   The allowlist of systems that this environment supports.
+    Valid values are `x86_64-linux`, `aarch64-linux`,
+    `x86_64-darwin`, and `aarch64-darwin`.
+    [`flox init`](./flox-init.md) automatically populates this list with the
+    current system type.
+    A user that attempts to pull an environment from FloxHub when their environment
+    isn't explicitly supported will be prompted whether to automatically add their
+    system to this list.
+    See [`flox-pull(1)`](./flox-pull.md) for more details.
+
+
+`allow.unfree`
+:   Allows packages with unfree licenses to be installed and appear in search
+    results.
+    The default is `false`.
+
+`allow.broken`
+:   Allows packages that are marked `broken` in the catalog to be installed and
+    appear in search results.
+    The default is `false`.
+
+`allow.licenses`
+:   An allowlist of software licenses to allow in search results in installs.
+    Valid entries are [SPDX Identifiers](https://spdx.org/licenses).
+
+`semver.allow-pre-releases`
+:   Whether to allow pre-release software for package installations.
+    The default is `false`.
+    Setting this value to `true` would allow a package version `4.2.0-pre`
+    rather than `4.1.9`.
+
+`cuda-detection`
+:   Whether to detect CUDA libraries and provide them to the environment.
+    The default is `true`.
+    When enabled, Flox will detect if you have an Nvidia device and attempt to
+    locate `libcuda` in well-known paths. Then it will symlink the libraries
+    into `.flox/lib` and add that path to `FLOX_ENV_LIB_DIRS`.
+
+# SEE ALSO
+[`flox-init(1)`](./flox-init.md),
+[`flox-install(1)`](./flox-install.md),
+[`flox-edit(1)`](./flox-edit.md)
```

Comment:
can do in followup if you prefer: since we flipped the feature flag, I think we can move all of these into `cli/flox/doc`

Reply:
done